framework/platform/bpl/uci/dhcp/bpl_dhcp.cpp

```diff
@@ -31,11 +34,12 @@ enum {
 };
 
 static const struct blobmsg_policy dhcp_event_policy[] = {
-    //TODO Remove: [DHCP_EVENT_ID] = { .name = "id", .type = BLOBMSG_TYPE_INT32 },
-    [DHCP_EVENT_OP]       = {.name = "op", .type = BLOBMSG_TYPE_STRING},
-    [DHCP_EVENT_MAC]      = {.name = "mac", .type = BLOBMSG_TYPE_STRING},
-    [DHCP_EVENT_IP]       = {.name = "ip", .type = BLOBMSG_TYPE_STRING},
-    [DHCP_EVENT_HOSTNAME] = {.name = "hostname", .type = BLOBMSG_TYPE_STRING}};
+    {.name = "id", .type = BLOBMSG_TYPE_INT32},       // [DHCP_EVENT_ID]
```

Comment:
Shouldn't this be part of a separate commit?

Reply:
It's a tiny C/C++ build issue fix. Not critical.

Reply:
This change is required because C++ (at least the GCC version used in UGW) doesn't support reordered or partial initializer lists... It not that big of a change. Expanded the macro and set default values for non initialized fields.